style.css

```diff
@@ -1 +1,193 @@
 /*  Add your styling here */
+body {
+  font-family: "Roboto", sans-serif;
+  width: 100%;
+  margin:0px;
+}
+
+.navbar {
+  display: none;
+}
+.nav {
+  display: none;
+}
+header {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  background-color: #7b3f00;
+ 
+}
+.header {
+  width: 60%;
+}
+.sideimg {
+  width: 40%;
+}
+.headerimg {
+  width: 150px;
+  padding: 25px 0px 35px 25px;
+}
+.hamburger {
+  width: 30px;
+  position: absolute;
+  top: 30px;
+  right: 30px;
+}
+.sitemoto {
+  text-align: right;
+  padding: 100px 30px 0px 25px;
+  font-size: 1.2rem;
+  color: lightgoldenrodyellow;
+}
+img {
+  transition: 1sec;
+}
+img:hover {
+  transform: scale(0.97);
+}
+
+section {
+  padding: 10px;
+}
+.maintext1 {
+  font-size: 2rem;
+  font-weight: 100;
+  margin-bottom: 20px;
+}
+.maintext2 {
+  font-size: 1rem;
+  font-weight: 100;
+  text-align: justify;
+}
+.imgsection {
+  width:100%;
+  
+}
+
+.bottomsection {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.produce {
+  width: 70%;
+ 
+}
+
+footer {
+  background-color: #7b3f00;
+  color: lightgoldenrodyellow;
+  
+}
+a:link {
+  color: lightgoldenrodyellow;
+  text-decoration: none;
+  margin-bottom: 30px;
+}
+a:visited {
+  color: lightgoldenrodyellow;
+}
+li {
+  padding: 0px 20px 0px 20px;
+  list-style-type: none;
+  font-size: 1rem;
+}
+.footer2 {
+  text-align: center;
+  padding-bottom: 35px;
+  margin-top: 0px;
+}
+@media only screen and (min-width: 540px) {
+  .bottomsection {
+    
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+  .alink {
+    width: 47%;
+    height: auto;
+    
+    
+  }
+  .produce{
+      width:100%;
+  }
+  footer {
+    background-color: #7b3f00;
+    color: lightgoldenrodyellow;
```

Comment:
Remember, inside media queries, you only need to include things that change! If your color stays the same, you can just declare it once (see line 79) and then you don't need to repeat it as it will apply always.